Makefile

```diff
@@ -11,7 +11,6 @@ GOTEST   := CGO_ENABLED=1 $(GO) test
 PACKAGES  := $$(go list ./... | grep -vE 'tests|cmd|vendor|pbmock')
 FILES    := $$(find . -name "*.go" | grep -vE "vendor")
 TOPDIRS  := $$(ls -d */ | grep -vE "vendor")
-SHELL    := /usr/bin/env bash
```

Comment:
Alpine doesn't have bash by default, and we don't use this too, so here just remove.